### DISCUSSION TITLE
Not working for me.

### DISCUSSION THREAD

**Discussion by Zenoka2004** [sigma67/spotify_to_ytmusic#114] [Q&A]
After i did everything as the README said.
C:\Users\user>spotify_to_ytmusic setup
'spotify_to_ytmusic' is not recognized as an internal or external command,
operable program or batch file.
Please help.

**Comment by sigma67** [comment]
Your PATH is not configured correctly for Python executables

**Comment by Zenoka2004** [comment]
![Screenshot 2024-09-26 133333](https://github.com/user-attachments/assets/f17958d1-c57a-4bc1-9523-e17d66228e6f)
I have already added it to Path and still not working.

**Reply by sigma67** [reply]
You need to add the bin path where your executables are located

https://stackoverflow.com/questions/35898734/pip-installs-packages-successfully-but-executables-not-found-from-command-line

**Reply by sigma67** [reply]
check the updated readme, it is easier when using pipx:

https://github.com/sigma67/spotify_to_ytmusic
